docs/_future/create_new_app.md

```diff
@@ -0,0 +1,765 @@
+
+> 💡 If you don't see the workspace you'd like to use in the list, you can `CTRL + C` out of the `slack run` command and run `slack auth login`. This will allow you to authenticate in your desired workspace to have it show up in the list for `slack run`.
+
+Once the app is successfully run, you'll see output in your Terminal to indicate it's running, similar to what you would see with any other Bolt for JavaScript app. You can search for the `⚡️ Bolt app is running! ⚡️` message to make sure that your app has successfully started up.
+
+With your app running, access your workspace and paste the URL from the the trigger you created in the previous step into a message in a public channel.
+
+> 💡 To make the trigger URL more widely accessible, we recommend saving the trigger as a channel bookmark for easy access.
+
+Send the message and click the "Run" button that appears. A modal will appear prompting you to enter information to request time off. To test your app properly, we recommend entering your own Slack username in the "Manager" field.
+
+![Request Time Off modal](../assets/take-your-time-modal.png "Request Time Off modal")
+
+Then, submit the form. You should receive a message from the app with information about the requested time off as well as an "Approve" and "Deny" button. Once either button is selected, you should then receive a message from the app notifying you of the status of your request.
+
+The full app flow can be seen here:
+![Request Time Off app](../assets/take-your-time-demo.gif "Request Time Off app")
+
+Now that we have a working instance of the app in your workspace, let's dive into its components and how it works!
+
+---
+### Workflows {#workflows}
+
+Workflows are a set of processing steps that are executed in order. Every workflow step consists of either a [built-in function](/bolt-js/future/built-in-functions) or [custom function](/bolt-js/future/custom-functions). Workflows can be configured to run without any user input, or they can wait for input via form before continuing.
+
+In the Request Time Off application, the Time Off Request workflow is defined in `manifest/workflows/time-off-request.js`. The general structure of a workflow is to first define it using the built in function `DefineWorkflow`:
+
+```js
+const { DefineWorkflow, Schema } = require('@slack/bolt');
+
+const TimeOffWorkflow = DefineWorkflow({
+  callback_id: 'time_off_request_wf',
+  title: 'Time Off Request Workflow',
+  input_parameters: {
+    properties: {
+      interactivity: {
+        type: Schema.slack.types.interactivity,
+      },
+    },
+    required: [],
+  },
+});
+
+module.exports = { TimeOffWorkflow };
+```
+
+In order to create a workflow that utilizes user input, such as one that uses a form, you'll need to pass in `interactivity` as one of the workflow's `input_parameters` under `properties`. This will allow the form to be interacted with.
```

Comment:
```suggestion
Here we want our workflow to take in user input, like form data. To do this we will need to pass in `interactivity` as one of the workflow's `input_parameters` under `properties`. This will allow the form to be interacted with.
```